### PR TITLE
DOCS: First pass at updating -B and adding -b for these 3 utilities

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
gpaddmirrors
gpmovemirrors
gprecoverseg

### CODE REVIEW

File: gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml [hunk 1]
```diff
@@ -79,11 +79,16 @@
           <pt>-a (do not prompt)</pt>
           <pd>Do not prompt the user for confirmation.</pd>
         </plentry>
+        <plentry>
+          <pt>-b <varname>segment_batch_size</varname></pt>
+          <pd>The maximum number of segments per host to operate on in parallel. Valid values are
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:84]: Please add the default value here. Something like:  "If not specified, the utility will start recovering up to 64 segments in parallel on each host. "

File: gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml [hunk 2]
```diff
@@ -79,11 +79,16 @@
...
+              <codeph>1</codeph> to <codeph>128</codeph>.</pd>
+        </plentry>
         <plentry>
           <pt>-B <varname>parallel_processes</varname></pt>
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:88]: -B parallel_processes --> -B batch_size

File: gpdb-doc/dita/utility_guide/ref/gpmovemirrors.xml [hunk 3]
```diff
@@ -32,11 +32,16 @@
     <section id="section4">
       <title>Options</title>
       <parml>
+        <plentry>
+          <pt>-b <varname>segment_batch_size</varname></pt>
+          <pd>The maximum number of segments per host to operate on in parallel. Valid values are
+              <codeph>1</codeph> to <codeph>128</codeph>.</pd>
+        </plentry>
         <plentry>
           <pt>-B <varname>parallel_processes</varname></pt>
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gpmovemirrors.xml:41]: -B parallel_processes --> -B batch_size

File: gpdb-doc/dita/utility_guide/ref/gpmovemirrors.xml [hunk 4]
```diff
@@ -32,11 +32,16 @@
     <section id="section4">
       <title>Options</title>
       <parml>
+        <plentry>
+          <pt>-b <varname>segment_batch_size</varname></pt>
+          <pd>The maximum number of segments per host to operate on in parallel. Valid values are
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gpmovemirrors.xml:37]: Please add the default value here. Something like: "If not specified, the utility will start recovering up to 64 segments in parallel on each host. "

File: gpdb-doc/dita/utility_guide/ref/gpmovemirrors.xml [hunk 5]
```diff
@@ -8,7 +8,7 @@
     <section id="section2">
       <title>Synopsis</title>
       <codeblock><b>gpmovemirrors</b> <b>-i</b> <varname>move_config_file</varname> [<b>-d</b> <varname>coordinator_data_directory</varname>] 
-          [<b>-l</b> <varname>logfile_directory</varname>] 
+          [<b>-l</b> <varname>logfile_directory</varname>] [<b>-b</b> <varname>segment_batch_size</varname>]
           [<b>-B</b> <varname>parallel_processes</varname>] [<b>-v</b>] [<b>--hba-hostnames</b> <varname>boolean</varname>]
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gpmovemirrors.xml:12]: -B parallel_processes --> -B batch_size

File: gpdb-doc/dita/utility_guide/ref/gpaddmirrors.xml [hunk 6]
```diff
@@ -92,11 +92,16 @@ Enter mirror segment data directory location 2 of 2 &gt; /gpdb/m2</codeblock>
                         configuration file with either <codeph>-m</codeph> or <codeph>-i</codeph> if
                         this option is used.</pd>
                 </plentry>
+                <plentry>
+                    <pt>-b <varname>segment_batch_size</varname></pt>
+                    <pd>The maximum number of segments per host to operate on in parallel. Valid
+                        values are <codeph>1</codeph> to <codeph>128</codeph>.</pd>
+                </plentry>
                 <plentry>
                     <pt>-B <varname>parallel_processes</varname></pt>
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gpaddmirrors.xml:101]: -B parallel_processes --> -B batch_size

File: gpdb-doc/dita/utility_guide/ref/gpaddmirrors.xml [hunk 7]
```diff
@@ -92,11 +92,16 @@ Enter mirror segment data directory location 2 of 2 &gt; /gpdb/m2</codeblock>
                         configuration file with either <codeph>-m</codeph> or <codeph>-i</codeph> if
                         this option is used.</pd>
                 </plentry>
+                <plentry>
+                    <pt>-b <varname>segment_batch_size</varname></pt>
+                    <pd>The maximum number of segments per host to operate on in parallel. Valid
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gpaddmirrors.xml:97]: Please add the default value here. Something like: "If not specified, the utility will start recovering up to 64 segments in parallel on each host. "

File: gpdb-doc/dita/utility_guide/ref/gpaddmirrors.xml [hunk 8]
```diff
@@ -10,11 +10,11 @@
         <section id="section2">
             <title>Synopsis</title>
             <codeblock><b>gpaddmirrors</b> [<b>-p</b> <varname>port_offset</varname>] [<b>-m</b> <varname>datadir_config_file </varname>[<b>-a</b>]] [<b>-s</b>] 
-   [<b>-d</b> <varname>coordinator_data_directory</varname>] [<b>-B</b> <varname>parallel_processes</varname>] [<b>-l</b> <varname>logfile_directory</varname>]
+   [<b>-d</b> <varname>coordinator_data_directory</varname>] [<b>-b</b> <varname>segment_batch_size</varname>] [<b>-B</b> <varname>parallel_processes</varname>] [<b>-l</b> <varname>logfile_directory</varname>]
    [<b>-v</b>] [<b>--hba-hostnames</b> <varname>boolean</varname>] 
 
 <b>gpaddmirrors</b> <b>-i</b> <varname>mirror_config_file</varname> [<b>-a</b>] [<b>-d</b> <varname>coordinator_data_directory</varname>]
-   [<b>-B</b> <varname>parallel_processes</varname>] [<b>-l</b> <varname>logfile_directory</varname>] [<b>-v</b>]
+   [<b>-b</b> <varname>segment_batch_size</varname>] [<b>-B</b> <varname>parallel_processes</varname>] [<b>-l</b> <varname>logfile_directory</varname>] [<b>-v</b>]
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gpaddmirrors.xml:17]: -B parallel_processes --> -B batch_size

File: gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml [hunk 9]
```diff
@@ -9,7 +9,7 @@
     <section id="section2">
       <title>Synopsis</title>
       <codeblock><b>gprecoverseg</b> [[<b>-p</b> <varname>new_recover_host</varname>[,...]] | <b>-i</b> <varname>recover_config_file</varname>] [<b>-d</b> <varname>master_data_directory</varname>] 
-             [<b>-B</b> <varname>parallel_processes</varname>] [<b>-F</b> [<b>-s</b>]] [<b>-a</b>] [<b>-q</b>] 
+             [<b>-b</b> <varname>segment_batch_size</varname>] [<b>-B</b> <varname>parallel_processes</varname>] [<b>-F</b> [<b>-s</b>]] [<b>-a</b>] [<b>-q</b>]
```
Comment (niralis13) [inline gpdb-doc/dita/utility_guide/ref/gprecoverseg.xml:12]: -B parallel_processes --> -B batch_size

**Review by niralis13 — CHANGES_REQUESTED** [review]

**Review by niralis13 — APPROVED** [review]

**Review by dyozie — APPROVED** [review]

**Pull request merged**